deep_quoridor/src/agents/alphazero/nn_evaluator.py

```diff
@@ -45,12 +45,16 @@ def evaluate(self, game: Quoridor):
         # Mask the policy to ignore invalid actions
         valid_actions = game.get_valid_actions()
         valid_action_indices = [self.action_encoder.action_to_index(action) for action in valid_actions]
-        policy_masked = np.zeros_like(unmasked_policy)
+        policy_masked = np.full_like(unmasked_policy, -1e32)
```

Comment:
`-1x10**32`? Could you add a comment explaining this here?

Reply:
Won't this give a sligth chance of taking an invalid action? probably they won't be chosen if there's any valid action with a reasonable probability, but in cases like we were debugging where the masked policies were all almost 0, it may end up chosing an invalid action.  I think what we would need to ensure is that some of the valid actions are > 0, we could take one randomly otherwise to avoid division by 0 but I think it's a red flag in the first place that the network is so confidently wrong